apps/api/src/app/workflows-v2/usecases/patch-step-data/patch-step.usecase.ts

```diff
@@ -38,26 +38,27 @@ export class PatchStepUsecase {
   }
 
   private async patchFieldsOnPersistedItems(command: PatchStepCommand, persistedItems: ValidNotificationWorkflow) {
-    if (command.name !== undefined) {
+    if (command.name !== undefined && command.name !== null) {
       await this.updateName(persistedItems, command);
     }
 
-    if (command.controlValues !== undefined) {
+    if (command.controlValues !== undefined && command.controlValues !== null) {
```

Comment:
not relevant to this pr:
make sure we check for nulls as well.

Reply:
need to revert this change, because it is incorrect due to the new FE-BE communication design decision from Friday.

Reply:
true, null is the way to communiate empty we want to update